pages/_app.js

```diff
@@ -1,10 +1,23 @@
 import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
 
 export default function App({ Component, pageProps }) {
   return (
     <>
-      <GlobalStyle />
-      <Component {...pageProps} />
+      <SWRConfig
+        value={{
+          fetcher: async (...args) => {
+            const response = await fetch(...args);
+            if (!response.ok) {
+              throw new Error(`Request with ${JSON.stringify(args)} failed.`);
+            }
+            return await response.json();
+          },
+        }}
+      >
```

Comment:
I'm not sure if there's going to be a use case for a `SWRConfig` in your app, since you're likely only going to fetch your data here, inside of `_app.js`. Right now, your database only returns one object, so there's no need to fetch it multiple times, since there are no other collections/etc. If you're planning on introducing multiple collections at some point, or you're going to be working with external APIs, then there's potentially a use case.